chainer/function_hooks/debug_print.py

```diff
@@ -22,7 +23,7 @@ class PrintHook(function_hook.FunctionHook):
     Chainer's library code.
 
     Attributes:
-        sep: Separator of print function.
+        sep: *(deprecated since v4.0.0)* Ignored.
```

Comment:
This is a list of attributes, not arguments, so is it better to keep `self.sep = sep` as is?

Reply:
Oh, I suspect it was intended to be argument documentation...

Reply:
Fixed.